public/app/features/alerting/unified/state/actions.ts

```diff
@@ -547,6 +547,7 @@ export const updateAlertManagerConfigAction = createAsyncThunk<void, UpdateAlert
     withAppEvents(
       withSerializedError(
         (async () => {
+          // TODO there must be a better way here than to dispatch another fetch as this causes re-rendering :(
```

Comment:
Not for this PR but AlertmanagerConfigAction is pretty straightforward so this is a good candidate to be refactored into RTK Query